### PR TITLE
Palindrome may 7 2

### PR DESCRIPTION
**Pull request by danielmarcano**
Hi, everyone.

Here's my submission for the Palindrome assignment.

For this one, I took full advantage of the "Fake It" technique, and used it to deliver the lowest possible amount of code needed to make my failing tests pass, until I could not fake the implementation anymore.

Any feedback would be very much appreciated!

### CODE REVIEW

File: 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts [hunk 1]
```diff
@@ -20,10 +20,10 @@ import { isPalindrome } from '.'
 
 describe('isPalindrome tests', () => {
   it('Returns a boolean', () => {
-    expect(typeof isPalindrome()).toBe('boolean');
+    expect(typeof isPalindrome('')).toBe('boolean');
   })
 
   it('Knows that "mom" is a palindrome', () => {
-    expect(typeof isPalindrome('mom')).toBe(true);
+    expect(isPalindrome('mom')).toBe(true);
```
Comment (n0rdlicht) [inline 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts]: **nitpick (non-blocking)**: `.toBe(true)` could be expressed as `.toBeTruthy()`

File: 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts [hunk 2]
```diff
@@ -1,5 +1,8 @@
 export function isPalindrome(text: string) {
-  if (text.toLowerCase().replace(' ', '') === 'mom') {
+  const parsedText = text.toLowerCase().replace(/ /g, '');
```
Comment (n0rdlicht) [inline 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts]: **suggestion (non-blocking)**: regex can be hard to read at times, alternative option: `.split(" ").join("")` or `.replace(' ', '')`

**Review by n0rdlicht — APPROVED** [review]
Nice solution, like the story the commits are telling about the process :) Added a few minor nitpicks/suggestions.

File: 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts (hunk 2 above)
Comment (danielmarcano) [reply 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.ts]: This is neat! Thank you. I will update the code as I prefer to write code as easier to understand and read as possible 😎

File: 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts (hunk 1 above)
Comment (danielmarcano) [reply 1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_2_Palindrome/src/index.spec.ts]: I always forget about this one! Thanks, I will update it

**Comment by danielmarcano** [comment]
@n0rdlicht thank you for the neat review! I have updated the code with your suggestions 🥳

**Pull request merged**